app/src/pages/common/settingsPage/notificationsTab/modals/addEditNotificationCaseModal/notificationCaseFormFields/notificationCaseFormFields.jsx

```diff
@@ -107,16 +109,25 @@ export class NotificationCaseFormFields extends Component {
   getAttributesConditionOptions = () => {
     const {
       intl: { formatMessage },
+      attributesValue,
+      resetSection,
     } = this.props;
+    const hasOneAttrOrLess = attributesValue.filter((attribute) => 'key' in attribute).length <= 1;
+
+    if (attributesValue.length === 1) {
+      resetSection([ATTRIBUTES_OPERATOR_FIELD_KEY]);
```

Comment:
This won't work if you already have a rule with 2 attributes and OR operator you can edit rule remove one attribute and according to the task we're expect operator should change to AND
`resetSection` reset to initial value

Reply:
You are absolutely right!